configs/override_rules.yaml

```diff
@@ -0,0 +1,17 @@
+---
+
+# Examples:
```

Comment:
Please provide full list of attributes we can change. And provide few more examples.